bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -484,6 +484,14 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <snow_thermal_cond_method>Jordan1991</snow_thermal_cond_method>
 <snow_thermal_cond_method phys="clm6_0">Sturm1997</snow_thermal_cond_method>
 
+<!-- TODO change clm6 to Jordan with answer changing tag for glacier mass ballance -->
+<snow_thermal_cond_glc_method>Jordan1991</snow_thermal_cond_glc_method>
+<snow_thermal_cond_glc_method phys="clm6_0">Sturm1997</snow_thermal_cond_glc_method>
+
+<!-- TODO change clm6 to Sturm with answer changing tag -->
+<snow_thermal_cond_lake_method>Jordan1991</snow_thermal_cond_lake_method>
+<snow_thermal_cond_lake_method phys="clm6_0">Jordan1991</snow_thermal_cond_lake_method>
+
```

Comment:
Oh, there is a missing part. This needs to go into the perl script. I forgot about that part...

Reply:
Thanks @ekluzek I think I've added what needs to be included now?  Please review.